methods.d.ts

```diff
@@ -41,6 +41,16 @@ type Ret<M extends keyof Telegram> = ReturnType<Telegram[M]>;
 /** Promisifies a given method signature */
 type P<M extends keyof Telegram> = (...args: Params<M>) => Promise<Ret<M>>;
 
+export interface Typegram<InputFile> {
+  Telegram: _Telegram<InputFile>;
+  TelegramP: { [M in keyof _Telegram<InputFile>]: P<M> };
+  Opts: {
+    [M in keyof _Telegram<InputFile>]: M extends (args?: infer O) => unknown
+      ? O
+      : {};
+  };
+}
+
```

Comment:
This prevents `telegraf` from re-exporting the `typegram` types, confer https://github.com/KnorpelSenf/typegram/pull/13#issuecomment-732136899.